src/index.html

```diff
@@ -11,7 +11,123 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <main>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label class="label" for="surname">
```

Comment:
you don't need for attribute here as you wrap input with label.
```suggestion
          <label class="label">
```